package.json

```diff
@@ -8,6 +8,8 @@
     "test": "karma start",
     "test-headless": "karma start headless.karma.conf.js",
     "debug": "karma start --single-run=false --debug --browser Chrome",
+    "copy-bundles": "copy-and-watch build/*.{json,js,woff,woff2,ttf} ../../../../Mount/eduid-front/build",
```

Comment:
Is there a better way to specify the target for the copy-and-watch script? If there is no apparent way we can wait to solve it until there are more users.

Reply:
Good question! I have now narrowed this down to only the bundles and seems to work for me.